nix/services/elasticsearch.nix

```diff
@@ -147,8 +147,11 @@ in
                 export ES_HOME ES_JAVA_OPTS ES_PATH_CONF
 
                 # Install plugins
-                rm -f "${config.dataDir}/plugins"
-                ln -sf ${esPlugins}/plugins "${config.dataDir}/plugins"
+                rm -rf "${config.dataDir}/plugins"
+                cp -rL ${esPlugins}/plugins "${config.dataDir}/plugins"
+                find "${config.dataDir}/plugins" -type f -exec chmod 0700 {} \;
```

Comment:
We don't really need to change the perms of the regular files, and I don't think we should make them executable.
```suggestion
```